src/main/java/org/gridsuite/modification/server/repositories/NetworkModificationRepository.java

```diff
@@ -394,15 +394,28 @@ public List<ModificationInfos> getModificationsInfos(@NonNull List<UUID> uuids)
         return uuids.stream().map(entities::get).filter(Objects::nonNull).map(this::getModificationInfos).toList();
     }
 
+    /**
+     * @param onlyCommonData if true, only returns the basic data common to all the modifications. If false, returns complete modifications
+     * @return the data from all the network modification contained in the composite modification sent as parameters
+     */
     @Transactional(readOnly = true)
-    public List<ModificationInfos> getCompositeModificationsInfos(@NonNull List<UUID> uuids) {
+    public List<ModificationInfos> getCompositeModificationsContentInfos(@NonNull List<UUID> uuids, boolean onlyCommonData) {
```

Comment:
I'm not sure this method should return wether a list of modification base infos or a modification full infos. You could rather implement 2 two methods